xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/Translations.xml

```diff
@@ -78,15 +78,15 @@ annotations.action.create.loaderror=Failed:
 annotations.action.create.error.unauthorized=You are not authorized to add annotations on this page. 
 annotations.action.create.error.unauthorizedguest=You are not authorized to add annotations on this page. Try to login first.
 annotations.action.create.helpmessage=To annotate a piece of text, select it and hit {0}.
-annotations.action.create.error.wrongsyntax=Annotations are not available for pages in XWiki/1.0 syntax.
+annotations.action.create.error.wrongsyntax=Annotations are not available for pages written in XWiki/1.0 syntax.
 annotations.action.create.error.notarget=Unspecified target (page) to create annotations for.
 
 annotations.action.view.hide.text=hide
 annotations.action.view.form.loaderror=Failed: 
 annotations.action.view.error.notfound=This annotation does not exist anymore. Please refresh the page for an updated view.
 annotations.action.view.thread=See thread
 
-annotations.altered.text=This annotation could not be displayed because the annotated text was not found in the page: 
+annotations.altered.text=This annotation could not be displayed because the annotated text was not found on the page: 
```

Comment:
So sure about this once. In sounds better since to me in this context. Would be nice to have another mentor's comment.

Reply:
I just think on sounds better